README.md

```diff
@@ -352,68 +397,42 @@ class MyInterceptor(ServerInterceptor):
 
 ### gRPC Compatibility
 
-In Connecpy, unlike connect-go, it is not possible to simultaneously support both gRPC and Connect RPC on the same server and port. In addition to it, Connecpy itself doesn't support gRPC. However, implementing a gRPC server using the same service code used for Connecpy server is feasible, as shown below. This is possible because the type signature of the service class in Connecpy is compatible with type signature gRPC framework requires.
-The example below uses [grpc.aio](https://grpc.github.io/grpc/python/grpc_asyncio.html) and there are in [example directory](example/README.md).
-
-```python
-# grpc_server.py
-import asyncio
-
-from grpc.aio import server
-
-import haberdasher_pb2_grpc
+In Connecpy, unlike connect-go, it is not possible to simultaneously support both gRPC and Connect RPC on the same server and port. Additionally, Connecpy itself doesn't support the gRPC wire protocol. 
 
-# same service.py as the one used in previous server.py
-from service import HaberdasherService
-
-host = "localhost:50051"
+The service signatures generated by Connecpy are different from those expected by gRPC:
+- **gRPC expects**: `async def MakeHat(self, request, context)`
+- **Connecpy generates**: `async def MakeHat(self, req: Size, ctx: ServiceContext) -> Hat`
 
+While the parameter names differ, there is limited syntactic compatibility when using positional arguments, since both signatures accept two arguments in the same order (message, context). However, this is not reliable because:
+1. The context types are incompatible (`ServiceContext` vs gRPC's `ServicerContext`)
+2. Keyword arguments will fail due to different parameter names
+3. The wire protocols are completely different
 
-async def main():
-    s = server()
-    haberdasher_pb2_grpc.add_HaberdasherServicer_to_server(HaberdasherService(), s)
-    bound_port = s.add_insecure_port(host)
-    print(f"localhost:{bound_port}")
-    await s.start()
-    await s.wait_for_termination()
-
-
-if __name__ == "__main__":
-    asyncio.run(main())
-```
+If you need to support both protocols, the recommended approach is to create an adapter layer:
 
 ```python
-# grpc_client.py
-import asyncio
-
-from grpc.aio import insecure_channel
-
-import haberdasher_pb2
-import haberdasher_pb2_grpc
-
-
-target = "localhost:50051"
-
-
-async def main():
-    channel = insecure_channel(target)
-    stub = haberdasher_pb2_grpc.HaberdasherStub(channel)
-    request = haberdasher_pb2.Size(inches=12)
-    response = await stub.MakeHat(request)
-    print(response)
-
-
-if __name__ == "__main__":
-    asyncio.run(main())
+class GrpcCompatAdapter:
```

Comment:
This example seems to be to serve a connecpy service with grpcio. I guess no one is going to do that, maybe we should reverse it?

Reply:
for now, I will remove this part entirely

Reply:
done https://github.com/i2y/connecpy/pull/71/commits/5bb8761326601da7c756d1936fa590c7c7aa0aa1